### ISSUE TITLE
Player pulling ragdolled self with grab mode exploit

### ISSUE THREAD

**Issue by Ratcha7** [naelstrof/KoboldKare#152]
I'm not sure if this is public knowledge already, but it's something I've been doing for a while now and has been a thing since patch 440. If a player ragdolls himself, uses grab mode to grab a part of his body and then continuously pulls it by scrolling forwards, it makes the player fly towards the direction he's pulling.

My words can't really give it justice, so I saved a clip of me showing it off: https://youtu.be/EBcZsblRtUw

In my opinion, this can be called an exploit, as it somewhat lowers the value of the jump pad upgrades and makes any sort of travel much faster, especially after mastering the movement mechanics.
However, not only has it saved me from a few times I got stuck, not only is it impossible to carry anything while doing this bug (To my knowledge), but it is also extremely fun to do.

I'm not sure what could be causing it though. Pulling other Kobolds including another player has never replicated the issue, but the moment one pulls itself it always works this way.

Of note is that the mission where you need to launch a Kobold way up in the air can be easily beaten with this bug.

Tested in patch 501

**Comment by DysWhite** [comment]
Yeah, I don't think this should be patched. The game is full of silly things like this, and this bug is very hard to do by accident. Plus, it's not that useful when bunnyhopping is often more controllable and therefor faster.

**Comment by naelstrof** [comment]
I'm going to leave this issue open for now, but not because it needs to be fixed, but because modding map creators should decide if this is a feature that's available.

Surf maps would be no fun if you could just grab yourself and fly through them, a simple world description that enables or disables this feature would be nice.
